### PR TITLE
[R2] Added Postman guide

### LINKED ISSUE #4619: R2 Storgage Implementation with S3
### Which Cloudflare product(s) does this pertain to?

R2

### Subject Matter

How Secret Access Key and Access Key ID be implemented in Postman

### Content Location

https://github.com/cloudflare/cloudflare-docs/blob/production/content/r2/platform/s3-compatibility/tokens.md

### Additional information

Please illustrate how Secret Access Key and Access Key ID to be implemented in authorization section in an API call?

### PR DESCRIPTION
**Pull request by mpint**
* Adds Postman guide to help users quickly get started on the platform (`content/r2/examples/postman.md`)
* Added supporting static images (`content/r2/static`)
closes #4619

### CODE REVIEW

File: content/r2/examples/postman.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Using R2 with Postman
+summary: Learn how to configure Postman to interact with R2.
+pcx-content-type: configuration
+weight: 1001
+layout: example
```
Comment (kristianfreeman) [inline content/r2/examples/postman.md:6]: @deadlypants1973 I noticed that this page is missing the TOC on the side, is that because of this field?

Comment (lukeed) [reply content/r2/examples/postman.md:6]: Yes :) All examples use this `layout`: https://github.com/cloudflare/cloudflare-docs/blob/production/layouts/_default/example.html Matches Workers' examples too

File: content/r2/examples/postman.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+Postman is an API platform that makes interacting with APIs easier. This guide will explain how to use Postman to make authenticated R2 requests to create a bucket, upload a new object, and then retrieve the object. The R2 [Postman collection](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290) includes a complete list of operations supported by the platform.
+
+## 1. Purchase R2
+
+Before getting started, you'll need an R2 subscription. [Learn more](https://developers.cloudflare.com/r2/get-started/#purchase-r2) about setting up your subscription.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
This guide assumes that you have made a Cloudflare account and [purchased R2](https://developers.cloudflare.com/r2/get-started/#purchase-r2).
```

File: content/r2/examples/postman.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## 2. Explore R2 in Postman
+
+Explore R2's publicly available [Postman collection](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290). The collection is organized into a `Buckets` folder for bucket-level operations and an `Objects` folder for object-level operations. Operations in the `Objects > Upload` folder allow for adding new objects to R2.
+
+## 3. Configure your R2 credentials
+
+Click on the `Cloudflare R2` collection and navigate to the `Variables` tab. Here, you can set variables within the R2 collection. They will be used to authenticate and interact with the R2 platform. Remember to always press Save after updating a variable.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
In the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation), select the **Cloudflare R2** collection and navigate to **Variables**. In **Variables**, you can set variables within the R2 collection. They will be used to authenticate and interact with the R2 platform. Remember to always select **Save** after updating a variable.
```

File: content/r2/examples/postman.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+![Postman Variables](/r2/static/postman-variables.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Select **Variables** in the Postman dashboard to continue with the example](/r2/static/postman-variables.png)
```

File: content/r2/examples/postman.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+To execute some basic operations, we must set `account-id`, `r2-access-key`, and `r2-secret-id`.
+
+Navigate to the [Cloudflare dashboard](https://dash.cloudflare.com/?account=r2) and select the account to use with R2. On the R2 overview page, copy your Cloudflare account ID in the from the right hand sidebar. Back in Postman, set the Current Value of `account-id` to your Cloudflare account ID and press Save.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
To do this:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/?account=r2).
2. In **Account Home**, select **R2**.
3. In **R2**, under **Manage R2 API Tokens** on the right side of the dashboard, copy your Cloudflare account ID.
4. Go back to the Postman dashboard.
5. Set the **CURRENT VALUE** of `account-id` to your Cloudflare account ID and select **Save**.
```

File: content/r2/examples/postman.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+Next, navigate back to the R2 overview page to set up your `r2-access-key` and `r2-secret-id`. On the right hand sidebar, click the _Manage R2 API Tokens_ link to create an R2 API Token called `Postman` and grant it the `Edit` permission.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
Next, generate an R2 API token:

1. Go back to the Cloudflare dashboard > **R2**.
2. On the right hand sidebar, select **Manage R2 API Tokens**.
3. Select **Create API token**.
4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the `Edit` permission.
```

File: content/r2/examples/postman.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+![Generate an R2 API token for Postman](/r2/static/postman-r2-api-token.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Generate an R2 API token for Postman by following the instructions above](/r2/static/postman-r2-api-token.png)
```

File: content/r2/examples/postman.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+Explore R2's publicly available [Postman collection](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290). The collection is organized into a `Buckets` folder for bucket-level operations and an `Objects` folder for object-level operations. Operations in the `Objects > Upload` folder allow for adding new objects to R2.
+
+## 3. Configure your R2 credentials
+
+Click on the `Cloudflare R2` collection and navigate to the `Variables` tab. Here, you can set variables within the R2 collection. They will be used to authenticate and interact with the R2 platform. Remember to always press Save after updating a variable.
+
+![Postman Variables](/r2/static/postman-variables.png)
+
+To execute some basic operations, we must set `account-id`, `r2-access-key`, and `r2-secret-id`.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
To execute basic operations, you must set `account-id`, `r2-access-key`, and `r2-secret-id` ___(noun needed to describe what these are) in __(where are you setting these)____.
```
Are these environment variables?

File: content/r2/examples/postman.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+Navigate to the [Cloudflare dashboard](https://dash.cloudflare.com/?account=r2) and select the account to use with R2. On the R2 overview page, copy your Cloudflare account ID in the from the right hand sidebar. Back in Postman, set the Current Value of `account-id` to your Cloudflare account ID and press Save.
+
+Next, navigate back to the R2 overview page to set up your `r2-access-key` and `r2-secret-id`. On the right hand sidebar, click the _Manage R2 API Tokens_ link to create an R2 API Token called `Postman` and grant it the `Edit` permission.
+
+![Generate an R2 API token for Postman](/r2/static/postman-r2-api-token.png)
+
+Guard this token closely. Anyone with this information can fully interact with all of your buckets.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]: Are you guarding the token? Or are you guarding the values of Access Key ID and Secret Access Key?

File: content/r2/examples/postman.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+
+Once created, copy the token’s `Access Key ID` value into Postman’s `r2-secret-id` variable and the `Secret Access Key` value into Postman’s `r2-access-key` variable and press Save.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
After you have created you API token in the Cloudflare dashboard:

1. Go back to the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation) > **Variables**.
2. Copy `Access Key ID` value from the Cloudflare dashboard and paste it into Postman’s `r2-secret-id` variable value and select **Save**.
3. Copy the `Secret Access Key` value value from the Cloudflare dashboard and paste into Postman’s `r2-access-key` variable value and select **Save**.
```

File: content/r2/examples/postman.md (hunk 8 above)
Comment (deadlypants1973) [reply content/r2/examples/postman.md]: @mpint please review the question and fill in the blanks.

File: content/r2/examples/postman.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+
+By now, you should have `account-id`, `r2-secret-id`, and `r2-access-key` set in Postman. To verify the token, navigate to the `ListBuckets` request and press `Send`. The Postman collection uses AWS SigV4 authentication to complete the handshake.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
By now, you should have `account-id`, `r2-secret-id`, and `r2-access-key` set in Postman. 

To verify the token:

1. In the Postman dashboard, select the **Cloudflare R2** folder dropdown arrow > **Buckets** folder dropdown arrow > **`GET`ListBuckets**. 
2. Select **Send**. 

The Postman collection uses AWS SigV4 authentication to complete the handshake.
```

File: content/r2/examples/postman.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+
+![ListBuckets in Postman](/r2/static/postman-r2-list-buckets.png)
+
+You should see a `200 OK` response with a list of existing buckets. If you receive an error, ensure your R2 subscription is active and Postman variables are saved correctly.
+
+## 4. Create a bucket
+
+Now, let’s have some fun. In your `Variables` tab, think of a fun name for a bucket and set it as the `r2-bucket` variable and press Save. Then, select the `CreateBucket` request and press `Send`.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
In the Postman dashboard:

1. Go to **Variables**. 
2. Set the `r2-bucket` variable value as the name of your R2 bucket and select **Save**. 
3. Select the **Cloudflare R2** folder dropdown arrow > **Buckets** folder dropdown arrow > **`PUT`CreateBucket** and select **Send**.
```

File: content/r2/examples/postman.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+
+![CreateBucket in Postman](/r2/static/postman-r2-create-bucket.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Follow the instructions above to navigate to CreateBucket in the Postman dashboard](/r2/static/postman-r2-create-bucket.png)
```

File: content/r2/examples/postman.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+You should see a `200 OK` response. If you run the `ListBuckets` request again, your bucket will appear in the list of results.
+
+## 5. Add an object
+
+A bucket isn’t very interesting without some stuff inside so let’s add an object. On the `Variables` tab, we set `r2-object` to `cat-pic.jpg` and press Save. Then, we select the `PutObject` request, and on the `Body` tab, we choose `binary` before attaching our favorite cat picture. Finally, we press `Send` to add a fluffy friend to R2.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
You will now add an object to your bucket:

1. Go to**Variables** in the Postman dashboard.
2. Set `r2-object` to `cat-pic.jpg` and select **Save**. 
3. Select **Cloudflare R2** folder dropdown arrow > **Objects** folder dropdown arrow > **Multipart** folder dropdown arrow > **`PUT`PutObject** and select **Send**. 
4. Go to **Body** and choose **binary** before attaching your cat picture. 
5. Select **Send** to add the cat picture to your R2 bucket.
```

File: content/r2/examples/postman.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+![PutObject in Postman](/r2/static/postman-r2-put-object.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Follow the instructions above to PutObject in the Postman dashboard](/r2/static/postman-r2-put-object.png)
```

File: content/r2/examples/postman.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+After a few seconds, you should receive a `200 OK` response.
+
+## 6. Get an object
+
+It only takes one more click to download our cat friend using the `GetObject` request.
+
+![GetObject in Postman](/r2/static/postman-r2-get-object.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Follow the instructions above to GetObject in the Postman dashboard](/r2/static/postman-r2-get-object.png)
```

File: content/r2/examples/postman.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+A bucket isn’t very interesting without some stuff inside so let’s add an object. On the `Variables` tab, we set `r2-object` to `cat-pic.jpg` and press Save. Then, we select the `PutObject` request, and on the `Body` tab, we choose `binary` before attaching our favorite cat picture. Finally, we press `Send` to add a fluffy friend to R2.
+
+![PutObject in Postman](/r2/static/postman-r2-put-object.png)
+
+After a few seconds, you should receive a `200 OK` response.
+
+## 6. Get an object
+
+It only takes one more click to download our cat friend using the `GetObject` request.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
To download your cat picture, select **Cloudflare R2** folder dropdown arrow > **Objects** folder dropdown arrow > **`GET`GetObject**.
```

Does the user need to press Send here or do something when they are on this page?

File: content/r2/examples/postman.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+
+![GetObject in Postman](/r2/static/postman-r2-get-object.png)
+
+You can explore the rest of the R2 Postman collection by playing with other operations. We’ll keep this collection up to date as we expand our feature set so stay tuned for updates.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
The R2 team will keep this collection up to date as we expand R2 features set. You can explore the rest of the R2 Postman collection by experimenting with other operations. 
```

**Review by deadlypants1973 — CHANGES_REQUESTED** [review]

File: content/r2/examples/postman.md (hunk 9 above)
Comment (mpint) [reply content/r2/examples/postman.md]: Yep, guarding of these values (but especially the secret) which comprise the token.

File: content/r2/examples/postman.md (hunk 17 above)
Comment (mpint) [reply content/r2/examples/postman.md]: Whoops, yep.  Looks like I ran out of steam here.  Will add the send command with your formatting.

File: content/r2/examples/postman.md [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+
+![Follow the instructions above to PutObject in the Postman dashboard](/r2/static/postman-r2-put-object.png)
+
+After a few seconds, you should receive a `200 OK` response.
+
+## 6. Get an object
+
+It only takes a few more more clicks to download our cat friend using the `GetObject` request.
+
+1. Select the **Cloudflare R2** folder dropdown arrow > **Objects** folder dropdown arrow > **`GET`GetObject** and select **Send**.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
1. Select the **Cloudflare R2** folder dropdown arrow > **Objects** folder dropdown arrow > **`GET`GetObject**.
2. Select **Send**.
```

File: content/r2/examples/postman.md [hunk 20]
```diff
@@ -0,0 +1,99 @@
...
+Next, generate an R2 API token:
+
+1. Go back to the Cloudflare dashboard > **R2**.
+2. On the right hand sidebar, select **Manage R2 API Tokens**.
+3. Select **Create API token**.
+4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the `Edit` permission.
+
+![Generate an R2 API token for Postman by following the instructions above](/r2/static/postman-r2-api-token.png)
+
+Guard this token closely. Anyone with this information can fully interact with all of your buckets.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
Guard this token and the **Access Key ID** and **Secret Access Key** closely. You will not be able to review these values again after finishing this step. Anyone with this information can fully interact with all of your buckets.
```

File: content/r2/examples/postman.md [hunk 21]
```diff
@@ -0,0 +1,99 @@
...
+3. In **R2**, under **Manage R2 API Tokens** on the right side of the dashboard, copy your Cloudflare account ID.
+4. Go back to the Postman dashboard.
+5. Set the **CURRENT VALUE** of `account-id` to your Cloudflare account ID and select **Save**.
+
+Next, generate an R2 API token:
+
+1. Go back to the Cloudflare dashboard > **R2**.
+2. On the right hand sidebar, select **Manage R2 API Tokens**.
+3. Select **Create API token**.
+4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the `Edit` permission.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the **Edit** permission.
```

File: content/r2/examples/postman.md (hunk 19 above)
Comment (deadlypants1973) [reply content/r2/examples/postman.md]: Do they need to do anything else on this screen other than pressing Send?

File: content/r2/examples/postman.md [hunk 22]
```diff
@@ -0,0 +1,99 @@
...
+
+Explore R2's publicly available [Postman collection](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290). The collection is organized into a `Buckets` folder for bucket-level operations and an `Objects` folder for object-level operations. Operations in the `Objects > Upload` folder allow for adding new objects to R2.
+
+## 3. Configure your R2 credentials
+
+In the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation), select the **Cloudflare R2** collection and navigate to the **Variables** tab. In **Variables**, you can set variables within the R2 collection. They will be used to authenticate and interact with the R2 platform. Remember to always select **Save** after updating a variable.
+
+![Select **Variables** in the Postman dashboard to continue with the example](/r2/static/postman-variables.png)
+
+To execute basic operations, you must set the `account-id`, `r2-access-key`, and `r2-secret-id` variables in the **Variables** tab.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
To execute basic operations, you must set the `account-id`, `r2-access-key`, and `r2-secret-id` variables in the Postman dashboard > **Variables**.
```

File: content/r2/examples/postman.md (hunk 19 above)
Comment (mpint) [reply content/r2/examples/postman.md]: Not unless we add "Enjoy the cute cat pic." 😆

File: content/r2/examples/postman.md [hunk 23]
```diff
@@ -0,0 +1,100 @@
...
+![Select **Variables** in the Postman dashboard to continue with the example](/r2/static/postman-variables.png)
+
+To execute basic operations, you must set the `account-id`, `r2-access-key`, and `r2-secret-id` variables in the Postman dashboard > **Variables**.
+
+To do this:
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/?account=r2).
+2. In **Account Home**, select **R2**.
+3. In **R2**, under **Manage R2 API Tokens** on the right side of the dashboard, copy your Cloudflare account ID.
+4. Go back to the Postman dashboard.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
4. Go back to the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation).
```

File: content/r2/examples/postman.md [hunk 24]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/?account=r2).
+2. In **Account Home**, select **R2**.
+3. In **R2**, under **Manage R2 API Tokens** on the right side of the dashboard, copy your Cloudflare account ID.
+4. Go back to the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation).
+5. Set the **CURRENT VALUE** of `account-id` to your Cloudflare account ID and select **Save**.
+
+Next, generate an R2 API token:
+
+1. Go back to the Cloudflare dashboard > **R2**.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
1. Go to the Cloudflare dashboard > **R2**.
```

File: content/r2/examples/postman.md [hunk 25]
```diff
@@ -0,0 +1,100 @@
...
+3. Select **Create API token**.
+4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the **Edit** permission.
+
+![Generate an R2 API token for Postman by following the instructions above](/r2/static/postman-r2-api-token.png)
+
+Guard this token and the **Access Key ID** and **Secret Access Key** closely. You will not be able to review these values again after finishing this step. Anyone with this information can fully interact with all of your buckets.
+
+After you have created you API token in the Cloudflare dashboard:
+
+1. Go back to the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation) > **Variables**.
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
1. Go to the [Postman dashboard](https://www.postman.com/cloudflare-r2/workspace/cloudflare-r2/collection/20913290-14ddd8d8-3212-490d-8647-88c9dc557659?action=share&creator=20913290&ctx=documentation) > **Variables**.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/r2/examples/postman.md [hunk 26]
```diff
@@ -0,0 +1,100 @@
...
+1. Go to the Cloudflare dashboard > **R2**.
+2. On the right hand sidebar, select **Manage R2 API Tokens**.
+3. Select **Create API token**.
+4. Name your token **Postman** by selecting the pencil icon next to the API name and grant it the **Edit** permission.
+
+![Generate an R2 API token for Postman by following the instructions above](/r2/static/postman-r2-api-token.png)
+
+Guard this token and the **Access Key ID** and **Secret Access Key** closely. You will not be able to review these values again after finishing this step. Anyone with this information can fully interact with all of your buckets.
+
+After you have created you API token in the Cloudflare dashboard:
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
After you have created your API token in the Cloudflare dashboard:
```

File: content/r2/examples/postman.md [hunk 27]
```diff
@@ -0,0 +1,100 @@
...
+By now, you should have `account-id`, `r2-secret-id`, and `r2-access-key` set in Postman.
+
+To verify the token:
+
+1. In the Postman dashboard, select the **Cloudflare R2** folder dropdown arrow > **Buckets** folder dropdown arrow > **`GET`ListBuckets**.
+2. Select **Send**.
+
+The Postman collection uses AWS SigV4 authentication to complete the handshake.
+
+![ListBuckets in Postman](/r2/static/postman-r2-list-buckets.png)
```
Comment (deadlypants1973) [inline content/r2/examples/postman.md]:
```suggestion
![Follow the instructions above to access ListBuckets in the Postman dashboard](/r2/static/postman-r2-list-buckets.png)
```

**Pull request merged**